launch/dual_ekf_navsat_example.launch.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2018 Open Source Robotics Foundation, Inc.
+# Copyright 2020 Fetullah Atas
```

Comment:
I don't think this is a substantial enough contribution to list your copyright here

Reply:
Adding the second copyright here isn’t necessarily about the how substantial the contribution is, it is there to help next user have an sense for tracking the history of the file.